src/nvim/eval.c

```diff
@@ -10399,6 +10399,24 @@ static dict_T *get_buffer_info(buf_T *buf)
   return dict;
 }
 
+/*
+ * "get_xdg_home(string)" function
+ */
```

Comment:
Use `///` for docstrings.  See: https://neovim.io/develop/style-guide.xml?showone=Function_Comments#Function_Comments